### PR TITLE
Refactor/clean

### PR DESCRIPTION
**Pull request by andrewowen**
Clean up some functions

### CODE REVIEW

File: src/app/RootStack.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  }
+);
+
+export const RootStack = createBottomTabNavigator(
+  {
+    "Life Counter": {
+      screen: LifeCounterNavigator,
+      navigationOptions: () => ({
+        tabBarIcon: () => <Icon name="logo-game-controller-a" size={25} />
+      })
```
Comment (Zanchee) [inline src/app/RootStack.js]: Maybe pull these `size` properties into a constant somewhere? Like `TabIconSize`

File: src/components/PlayerComponent.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import PlayerLifeButton from './PlayerLifeButton';
+
+export class PlayerComponent extends Component {
+  state = {
+    currentLife: 40,
+    minLife: 0
```
Comment (Zanchee) [inline src/components/PlayerComponent.js]: Same thing here, maybe make this `40` a constant.

File: src/components/PlayerComponent.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  };
+  addLife = () => {
+    this.setState((prevState, props) => ({
```
Comment (Zanchee) [inline src/components/PlayerComponent.js]: You can leave `props` out from the parameters if you aren't using it. (Either way, not a big deal!)

Also, you could do `this.setState({currentLife: this.state.currentLife - 1})`

File: src/components/PlayerComponent.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            fontFamily: 'SourceCodePro-Bold'
+          }}
+        >
+          <View
+            style={{
+              alignContent: 'center',
+              justifyContent: 'center',
+              alignContent: 'center'
+            }}
+          >
```
Comment (Zanchee) [inline src/components/PlayerComponent.js]: Pull these inline styles into the `styles` stylesheet?

File: src/components/PlayerComponent.js (hunk 2 above)
Comment (andrewowen) [reply src/components/PlayerComponent.js]: I'm fine with this being declared here as it is a state object

File: src/components/PlayerComponent.js (hunk 4 above)
Comment (andrewowen) [reply src/components/PlayerComponent.js]: these will be changed in a future push

**Pull request merged**